classes/class-hmbkp-path.php

```diff
@@ -62,20 +64,17 @@ private function __wakeup() {}
 	/**
 	 * Returns the *Singleton* instance of this class.
 	 *
-	 * @staticvar HMBKP_PAth $instance The *Singleton* instances of this class.
+	 * @staticvar HMBKP_Path $instance The *Singleton* instances of this class.
 	 *
 	 * @return HMBKP_Path The *Singleton* instance.
 	 */
 	public static function get_instance() {
 
-		static $instance = null;
```

Comment:
Yeah, I'd assume that too, I copied the code from here though: 

http://www.phptherightway.com/pages/Design-Patterns.html#singleton

Maybe it's "Late `static` binding"? although I don't totally have my head wrapped around that...

Maybe @joehoyle can provide insight.